docs/data/sdks/android-kotlin/index.md

```diff
@@ -293,3 +293,494 @@ amplitude.add(
     new SegmentDestinationPlugin(this, SEGMENT_WRITE_KEY)
 )
 ```
+
+## Advanced Topic
+
+### User sessions
+
+A session on Android is a period of time that a user has the app in the foreground.
+
+Amplitude groups events together by session. Events that are logged within the same session have the same `session_id`. Sessions are handled automatically so you don't have to manually call `startSession()` or `endSession()`.
+
+You can adjust the time window for which sessions are extended.
+
+=== "Java"
+
+    ```
+    amplitude = AmplitudeKt.Amplitude(AMPLITUDE_API_KEY, getApplicationContext(), configuration -> {
+        configuration.setMinTimeBetweenSessionsMillis(1000);
+        return Unit.INSTANCE;
+    });
+    ```
+=== "Kotlin"
+
+    ```kotlin
+    amplitude = Amplitude(
+            Configuration(
+                apiKey = AMPLITUDE_API_KEY,
+                context = applicationContext,
+                minTimeBetweenSessionsMillis = 10000
+            )
+        )
+    ```
+
+By default, Amplitude will auto send the '[Amplitude] Start Session' and '[Amplitude] End Session'. Even though these events aren't sent, sessions are still tracked by using `session_id`.
```

Comment:
This seems contradictory? I also think it might be missing a word, maybe _events?_ -- "the '[Amplitude] Start Session' and '[Amplitude] End Session'. _events_"

Reply:
More explanation:
Logging the session events are default behavior, but customers can disable this. So if the customer disables the trackingSessionEvents, they will not receive `'[Amplitude] Start Session' and '[Amplitude] End Session'` events. 
But each event will still have a session_id value unless there has a setting to force the session_id = -1.

Reply:
Let me know if you have better words to clarify those all.